src/libraries/System.ServiceProcess.ServiceController/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,75 @@
+    <Target>T:System.ServiceProcess.ServiceBase</Target>
+    <Left>lib/netstandard2.0/System.ServiceProcess.ServiceController.dll</Left>
+    <Right>lib/net461/System.ServiceProcess.ServiceController.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.ServiceProcess.ServiceController</Target>
+    <Left>lib/netstandard2.0/System.ServiceProcess.ServiceController.dll</Left>
+    <Right>lib/net461/System.ServiceProcess.ServiceController.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.ServiceProcess.ServiceControllerStatus</Target>
+    <Left>lib/netstandard2.0/System.ServiceProcess.ServiceController.dll</Left>
+    <Right>lib/net461/System.ServiceProcess.ServiceController.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.ServiceProcess.ServiceProcessDescriptionAttribute</Target>
+    <Left>lib/netstandard2.0/System.ServiceProcess.ServiceController.dll</Left>
+    <Right>lib/net461/System.ServiceProcess.ServiceController.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.ServiceProcess.ServiceStartMode</Target>
+    <Left>lib/netstandard2.0/System.ServiceProcess.ServiceController.dll</Left>
+    <Right>lib/net461/System.ServiceProcess.ServiceController.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.ServiceProcess.ServiceType</Target>
+    <Left>lib/netstandard2.0/System.ServiceProcess.ServiceController.dll</Left>
+    <Right>lib/net461/System.ServiceProcess.ServiceController.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.ServiceProcess.SessionChangeDescription</Target>
+    <Left>lib/netstandard2.0/System.ServiceProcess.ServiceController.dll</Left>
+    <Right>lib/net461/System.ServiceProcess.ServiceController.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.ServiceProcess.SessionChangeReason</Target>
+    <Left>lib/netstandard2.0/System.ServiceProcess.ServiceController.dll</Left>
+    <Right>lib/net461/System.ServiceProcess.ServiceController.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.ServiceProcess.TimeoutException</Target>
+    <Left>lib/netstandard2.0/System.ServiceProcess.ServiceController.dll</Left>
+    <Right>lib/net461/System.ServiceProcess.ServiceController.dll</Right>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV006</DiagnosticId>
+    <Target>.NETStandard,Version=v1.4</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV006</DiagnosticId>
+    <Target>.NETStandard,Version=v1.5</Target>
+  </Suppression>
+  <Suppression>
+    <DiagnosticId>PKV007</DiagnosticId>
+    <Target>.NETStandard,Version=v1.5-win</Target>
```

Comment:
@safern @Anipik is this diagnostic redundant to the RID-less one above?  IOW if a package dropped support for a TFM completely doesn't that also mean it dropped support for a specific RID on that TFM?

Reply:
Yes, I think we should just emit the ridless one. 

@Anipik thoughts?

Reply:
yes agreed i can take a look at this one.

Reply:
Filed https://github.com/dotnet/sdk/issues/19517.  I'm actually a little less certain about this one after writing that up.